broker/pkg/storage/timeline/timeline.go

```diff
@@ -47,7 +47,7 @@ type Repository interface {
 	// COUNT messages BY TimelineID, RANGE and LockConsumerID is not empty (count waiting status)
 	CountByRangeWaiting(ctx context.Context, timelineID []byte, a, b uint64) uint64
 	// SELECT messages by TimelineID, LockConsumerID (when consumer restarts)
-	SelectByConsumer(ctx context.Context, timelineID []byte, consumerID []byte, buckets domain.BucketRange, maxTimestamp uint64) []timeline.Message
+	SelectByConsumer(ctx context.Context, timelineID []byte, consumerID []byte, buckets domain.BucketRange, limit int, maxTimestamp uint64) ([]timeline.Message, []errors.MessageIDTuple)
```

Comment:
`SelectByConsumer` should return `([]timeline.Lease, bool, error)` to remain consistent with method `GetAndLease`